bin/compile

```diff
@@ -63,17 +63,21 @@ PATH="${PATH}:${CRYSTAL_DIR}/bin"
 # Build the project
 cd $BUILD_DIR
 
-start "Installing Dependencies"
-  crystal deps --production
-finished
-
 NO_DEBUG_IF_AVAILABLE=`crystal build --help | grep -q -- --no-debug && echo -n --no-debug`
 
 if [ -f app.cr ]; then
+  start "Installing Dependencies (crystal deps)"
+    crystal deps --production
```

Comment:
`crystal deps` is always just an alias for `shards` now so...?

Reply:
you are right, for once I thought it was about old deps mechanism which I was near to delete later (but that has already happened..)